src/Backend/Modules/Slideshows/Actions/Add.php

```diff
@@ -33,6 +35,8 @@ private function handleForm()
         // create elements
         $txtTitle = $this->frm->addText('title', null, null, 'form-control title', 'form-control danger title');
 
+        $template = $this->frm->addDropdown('template', Model::getPossibleTemplates());
```

Comment:
you should only show/add this dropdown if there are more than 1 possible template, just like we do in the Content Blocks module